### ISSUE TITLE
[idea] Would you put some of your board photos and a readme ?

### ISSUE THREAD

**Issue by marshel** [vanBassum/RZ_EasyFPGA#1]
Hi, I believe you had a similar idea as mine. 

I bought a RZ Easy_FPGA, saw ben eaters video and wanted to build my machine on it.

Now I'm in a learning curve and my purpose is to build a MSX 2+ (Z80) compatible machine.
but 1st I need to learn about FGPAs and how to build/sintetize something in it.

Best regards,

**Comment by marshel** [comment]
P.S. some docs about mine version are here as well: https://github.com/dualvim/KitEasyFPGA/tree/Contribs_Dualvim

**Comment by vanBassum** [comment]
Hey, I didn't notice your issue, sorry for the late reply.

This repo is just to store some tests I did myself. My knowledge of VHDL isn't that great anyway. However, I could add something if you would like. If I recall correctly, the beneater example should work, but I remember that it isn't 100% the same.  I can try to figure out what I did back then and add a simple readme.
Now that I think of it, I recall creating a very basic assembler in C#, that converted asm commands to hex. This hex could be inserted into the ram.

It might be fun to 'complete' this project by adding a simple blinking LED program. That runs on the beneater CPU implemented on the RZ devboard. The readme could be a simple instruction to get this blinking LED thing to work.

About the Z80, that is a project from: http://searle.x10host.com/Multicomp/index.html
I just made it work on the rz devboard.
